README.md

```
@@ -12,20 +12,37 @@ To determine the SharpHound version compatible with a deployed BloodHound CE ins
 
 Please refer to the [SharpHound section](https://bloodhound.specterops.io/collect-data/ce-collection/sharphound), part of the [BloodHound Community Edition documentation](https://bloodhound.specterops.io/home). 
 
-## Compile Instructions
+## Compilation Instructions
 
-To build this project, use .net 5.0 and run the following:
+To build this project, using a .NET SDK and run the following:
 
 ```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)